locales/en/login.json

```diff
@@ -9,6 +9,9 @@
     "continue-apple": "Continue with Apple",
     "other-options": "Other Login Options",
     "email-sent": "We just sent an email to <strong>{{email}}</strong>",
+    "check-spam": "Didn't get anything? Please check your promotion or spam",
```

Comment:
This message needs better wordings. E.g. "Didn't receive verification email? Click below to resend." or something like that

Reply:
```
Didn't receive the verification email? Check your spam inbox

Resend verification email in 3 sec...
```